torch/_inductor/codegen/cpp.py

```diff
@@ -2246,6 +2251,14 @@ def gen_loop_with_reduction(
 
                     gen_loop_at(_loop_nest, depth)
 
+                    if loop.is_reduction and not in_reduction:
+                        if loop.parallel:
+                            if kernel.local_welford_reduction_stores:
+                                code.splice(kernel.local_welford_reduction_stores)
+                        else:
+                            if kernel.welford_reduction_stores:
+                                code.splice(kernel.welford_reduction_stores)
```

Comment:
Can we move it to a helper function?

Reply:
I reused `local_reduction_stores `and `reduction_suffix`, deleted these lines.